pod/main/templates/navbar.html

```diff
@@ -7,7 +7,7 @@
     <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
       <i data-feather="menu" aria-hidden="true"></i>
     </button>
-    <a class="navbar-brand mr-0 mr-sm-1" href="/"><img src="{% static LOGO_SITE %}" height="48" alt="logo"/><strong>{{ TITLE_SITE }}</strong></a>
+    <a class="navbar-brand mr-0 mr-sm-1" href="/"><img src="{% static LOGO_SITE %}" height="48" alt="" role="presentation"/><strong>{{ TITLE_SITE }}</strong></a>
```

Comment:
@Badatos why remove content from alt attribute ?

Reply:
When alt doesn't add usefull content to blind people, it is better to set an empty "alt", with role="presentation".
Here, it is not usefull to tell only "logo".

Reply:
see more here : https://davidwalsh.name/accessibility-tip-empty-alt-attributes

Reply:
Check this : https://stackoverflow.com/questions/23526791/when-is-it-acceptable-to-leave-the-alt-text-empty-on-an-html-image-if-ever#answer-23526792

Reply:
le alt n'est pas toujours obligatoire ou recommandé, par contre, pour un logo, on peut spécifier ce qu'il représente ou préciser "logo de l'université Côte d'azur" qu'en pensez-vous ?

Reply:
![image](https://user-images.githubusercontent.com/36244941/99047131-05f10080-2594-11eb-95f2-6053a72f3b4e.png)
@ptitloup  je suis d'accord

Reply:
Il est conseiller de préciser un alt si une image est cliquable, par contre, le alt n'est pas nécessaire si l'image est suivi d'un texte la décrivant... bref, visiblement tout le monde a raison ! :D

Reply:
Ici, ce serait plutot "logo de Pod", car le logo de l'établissement est ailleurs. Mais est-ce vraiment utile qu'un mal voyant sache que c'est le logo du logiciel ici ? Le plus important, c'est le texte qui suit : "TITLE_SITE".

Reply:
du coup, je pense que ca mange pas de pain de rajouter un alt... @Badatos est-ce possible de mettre le alt="Logo {{TITLE_SITE}}" ou un truc comme ca ?

Reply:
Merci !

Reply:
Par contre je suis d'accord qu'on pourrait ajouter un alt personnalisé sur le logo de l'établissement. Mais attention à ne pas être redondant avec le nom de l'établissement qui suit.

Reply:
alt="Logo {{TITLE_SITE}}"  serait contre productif, car "TITLE_SITE" est dans le lien juste après. La synthèse vocale va juste bégailler "Logo TITLE_SITE TITLE_SITE".
(N'hésitez pas à activer "Voice Over" sur Mac ou le narrateur Microsoft sur Windows.)

Reply:
en effet, mon correspondant accessibilité précise "dans ce cas, je le mettrai pas. Ca apporte rien pour les usagers..."

Reply:
Je propose donc de laisser telquel